src/test/java/com/qaprosoft/carina/demo/WebLocalizationSample.java

```diff
@@ -0,0 +1,32 @@
+package com.qaprosoft.carina.demo;
+
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.qaprosoft.carina.core.foundation.utils.ownership.MethodOwner;
+import com.qaprosoft.carina.core.foundation.utils.resources.L10N;
+import com.qaprosoft.carina.demo.gui.pages.localizationSample.WikipediaHomePage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * This sample shows how create Web Localization test with Resource Bundle.
+ *
+ * @author qpsdemo
+ */
+
+public class WebLocalizationSample extends AbstractTest {
+
+    @Test
+    @MethodOwner(owner = "qpsdemo")
+    public void testLanguages() throws Exception {
```

Comment:
throws Exception - seems unused. if necessary you should refactorr methods inside